server/src/main/java/org/apache/cloudstack/backup/BackupManagerImpl.java

```diff
@@ -1012,6 +1006,7 @@ protected void runInContext() {
                 }
                 for (final DataCenter dataCenter : dataCenterDao.listAllZones()) {
                     if (dataCenter == null || isDisabled(dataCenter.getId())) {
+                        LOG.warn(String.format("Backup Sync Task is not enabled in zone [%s]. Skipping this zone!", dataCenter == null ? "NULL Zone!" : dataCenter.getId()));
```

Comment:
why an `error` message for every zone, if its not there? we are looking for only one, so `info` er even `debug` should be enough.

Reply:
Done.